src/main/java/decodes/tsdb/groupedit/TsListPanel.java

```diff
@@ -215,7 +230,24 @@ public void refresh()
 	{
 		tsListSelectPanel.refreshTSIDList();
 	}
-	
+
+	@Override
+	public void plot()
+	{
+		try
+		{
+			TimeSeriesDAI dao = theDb.makeTimeSeriesDAO();
+			final TimeSeriesIdentifier[] tsids = tsListSelectPanel.getSelectedTSIDs();
+			TimeSeriesChartFrame f = new TimeSeriesChartFrame(dao,tsids);
```

Comment:
I suspect this should take the TimeseriesDb and not the TimeseriesDao, this would make it easier to keep the chart frame in memory and reuse it by just passing in new tsids. 

Or the chart shouldn't be responsible for loading the the timeseries and that work should happen in... but in a swingworker.

Reply:
Yeah, I'm suggesting it be given list of TimeSEriesLine, and TsListController/Panel would handle actually retrieving the data.